scripts/README.md

```diff
@@ -0,0 +1,39 @@
+## Miscellaneous scripts for working on mathlib
+
+This directory contains miscellaneous scripts that are useful for working on or with mathlib.
+When adding a new script, please make sure to document it here, so other readers have a chance
+to learn about it as well!
+
+**Current scripts and their purpose.**
+TODO: add real content for all these entries!
+
+Part of mathlib
+- `polyrith_sage.py`, `polyrith_sage_helper.py`
+
+Part of CI workflow
+- `mk_all.lean`
+- `lint-style.py` and `lint-style.lean`, `print-style-errors.sh`
+- `lint-bib.sh`
+- `yaml_check.py`, `check-yaml.lean`
+
+Managing and tracking technical debt
+- `update_nolints_CI.sh` (maintenance, runs once a week)
+- `technical-debt-metrics.sh`
+- `init_creation.sh` (supposed to be run manually; might be removed soon?)
+
+- `create-adaptation-pr.sh`, `lean-pr-testing-comments.sh`
```

Comment:
`create-adaptation-pr.sh` implements some of the steps in the workflow described at https://leanprover-community.github.io/contribute/tags_and_branches.html#mathlib-nightly-and-bump-branches
Specifically, it will:
- merge `master` into `bump/v4.x.y`
- create a new branch off of `bump/v4.x.y`
- merge `nightly-testing` into the new branch
- open a PR to merge the new branch back into `bump/v4.x.y`
- announce the PR on zulip

If there are merge conflicts, it pauses and asks for help from the human driver.

Reply:
I've moved this into the README